packages/core/src/tracing/trace.ts

```diff
@@ -253,13 +250,14 @@ function createChildSpanOrTransaction({
       traceId,
       parentSpanId,
       parentSampled: sampled,
+      // TODO: We should probably only set this if we know we are continuing a trace
+      // For now, there isn't really a way to know this from in here, though...
```

Comment:
with "this" are you referring to `spanContext`? Or should this be moved to the `if(dsc)` part below?

Reply:
ah wait, sorry, I did not move this comment along correctly with code I moved 😅 yeah it belongs to below!